Community-Aktivitäten/README.md

```diff
@@ -165,6 +165,46 @@ Dauer: 2 Stunden (bei Bedarf auch gerne mehr)
 
 Ausstattung: großer Screen/Beamer, Internet
 
+## Building your career: Ask anything
```

Comment:
@ixchelruiz Thank you for your proposal. To allow further discussion about Hendrik's proposals I have set this PR to draft state. Please decide about the final text and when done set back to "Ready for review" state. Thanks!

Reply:
If you have already prepared them then please post them ASAP, because the deadline for the printed brochures is near, and if you post too late we have no chance to do marketing for your events.